lib/discourse_chat/provider/slack/slack_message.rb

```diff
@@ -36,8 +37,14 @@ def text
         text = parts.length > 1 ? parts[1] : parts[0]
 
         if parts[0].start_with?('@')
-          user = @transcript.users.find { |u| u["id"] == parts[0].gsub('@', '') }
-          next "@#{user['name']}"
+          user_id = parts[0].gsub('@', '')
+          user = @transcript.users[user_id]
```

Comment:
Trivial but you could use the `if user = @transcript.users[user_id]` syntax here and save a line.